src/libs/Library.lua

```diff
@@ -50,21 +49,45 @@ end
 --[[
     Converts seconds in to text friedly format
 ]]
-function CC_Libs.SecondsToReadibleFormat(seconds)
-    local timeMinutes = seconds     / 60
+function CC_Libs.SecondsToReadibleFormat(totalSeconds)
```

Comment:
Changed how the timer countdown displays.  It will now say the full hours, minutes, and seconds remaining,